tutorials/multi/1_graph_classification.py

```diff
@@ -0,0 +1,241 @@
+tensors (e.g. labels) on the same GPU:
+"""
+
+"""
+import torch
+
+# Use the first GPU
+device = torch.device("cuda:0")
+model = model.to(device)
+graph = graph.to(device)
+labels = labels.to(device)
+"""
+
+###############################################################################
+# The node and edge features in the graphs, if any, will also be on the GPU.
+# After that, the forward computation, backward computation and parameter
+# update will take place on the GPU. For graph classification, this repeats
+# for each minibatch gradient descent.
+#
+# Using multiple GPUs allows performing more computation per unit of time. It
+# is like having a team work together, where each GPU is a team member. We need
+# to distribute the computation workload across GPUs and let them synchronize
+# the efforts regularly. PyTorch provides convenient APIs for this task with
+# multiple processes, one per GPU, and we can use them in conjunction with DGL.
+#
+# Intuitively, we can distribute the workload along the dimension of data. This
+# allows multiple GPUs to perform the forward and backward computation of
+# multiple gradient descents in parallel. To distribute a dataset across
+# multiple GPUs, we need to partition it into multiple mutually exclusive
+# subsets of a similar size, one per GPU. We need to repeat the random
+# partition every epoch to guarantee randomness. We can use
+# :func:`~dgl.dataloading.pytorch.GraphDataLoader`, which wraps some PyTorch 
+# APIs and does the job for graph classification in data loading.
+#
+# Once all GPUs have finished the backward computation for its minibatch,
+# we need to synchronize the model parameter update across them. Specifically,
+# this involves collecting gradients from all GPUs, averaging them and updating
+# the model parameters on each GPU. We can wrap a PyTorch model with
+# :func:`~torch.nn.parallel.DistributedDataParallel` so that the model
+# parameter update will invoke gradient synchronization first under the hood.
+#
+# That’s the core behind this tutorial. We will explore it in more detail with
+# a complete example below.
+#
+# Distributed Process Group Initialization
+# ----------------------------------------
+#
+# For communication between multiple processes in multi-gpu training, we need
+# to start the distributed backend at the beginning of each process. We use
+# `world_size` to refer to the number of processes and `rank` to refer to the
+# process ID, which should be an integer from `0` to `world_size - 1`.
+# 
+
+import torch.distributed as dist
+
+def init_process_group(world_size, rank):
+    dist.init_process_group(
+        backend='nccl',
+        init_method='tcp://127.0.0.1:12345',
+        world_size=world_size,
+        rank=rank)
+
+###############################################################################
+# Data Loader Preparation
+# -----------------------
+#
+# Split the dataset into training, validation and test subsets. In dataset
```

Comment:
The first sentence seems incomplete

Reply:
It's an imperative sentence. See https://literarydevices.net/imperative-sentence/#:~:text=An%20imperative%20sentence%20is%20a,exclamation%20mark%20or%20a%20period.

Reply:
I know it's valid in grammar. But I'm wondering why you change the way of using `we` as the subjective here. I personally feel imperative sentence here breaks the whole flow in the tutorial, the tone here is harder than the other part

Reply:
How about adding "We" at the beginning?